pkg/hydrator/nexpose.go

```diff
@@ -301,6 +302,11 @@ func (n *NexposeClient) makeNexposeRequest(queryParams map[string]string, pathFr
 	}
 	defer res.Body.Close()
 
+	if res.StatusCode != 200 {
+		// Nexpose is down?  Nexpose unexpectedly gave 404?
+		return nil, fmt.Errorf("Nexpose unexpectedly returned non-200 response code: %d", res.StatusCode)
```

Comment:
suggestion: For easier debugging, we often include the response body from the remote system so we can see any additional details about the failure that it sent.